router/core/context.go

```diff
@@ -20,6 +20,7 @@ type key string
 
 const requestContextKey = key("request")
 const subgraphsContextKey = key("subgraphs")
+const baseAttributesContextKey = key("baseOtelAttributes")
```

Comment:
Should it be "baseOTELAttributes"?

Reply:
We use lowercase everywhere and also the library writes it likes that.